scripts/buildkite/release/generate-trigger.sh

```diff
@@ -7,6 +7,13 @@ VERSION=$(buildkite-agent meta-data get "release-version")
 BRANCH=$(buildkite-agent meta-data get "release-candidate-branch")
 CABAL=$(buildkite-agent meta-data get "release-cabal-version")
 TEST_RC=$(buildkite-agent meta-data get "test-rc")
+BASE_BUILD=$(buildkite-agent meta-data get "base-build")
+
+if [ "$TEST_RC" == "TURE" ]; then
```

Comment:
```suggestion
if [ "$TEST_RC" == "TRUE" ]; then
```

🤔 

But see here for an alternative design: https://hackage.haskell.org/package/acme-php-0.0.3/docs/Prelude-PHP.html#t:Bool

Reply:
:fearful: